src/subscription_manager/managercli.py

```diff
@@ -76,6 +76,7 @@
 from rhsmlib.services import config, attach, products, unregister, entitlement, register
 from rhsmlib.services import exceptions
 from subscription_manager import syspurposelib
+from syspurpose.files import SyncedStore
```

Comment:
The syspurpose package is not available on some platforms. E.g. SuSE. Thus I would not import the `SyncedStore` like this.